charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -22,8 +22,8 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 40.2.0
-appVersion: 0.59.1
+version: 40.2.1
```

Comment:
this should be a minor

Reply:
@dotdc do you mean major? as in 41.0.0? no crds are changing.

Reply:
Minor, the middle number

Reply:
@dotdc done